src/bundle/Resources/views/layout.html.twig

```diff
@@ -48,7 +48,32 @@
             </div>
         </div>
         <div class="container-fluid ez-main-container">
-            {% block content %}{% endblock %}
+            {% block content %}
+
+                {% block sidebar %}
+                    <div class="col-sm-1 bg-dark pt-4 ez-side-menu">
+                        <a class="btn btn-dark btn-block" href="{{ path('ezplatform.search') }}">
+                            <svg class="ez-icon ez-icon-search">
+                                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#search"></use>
+                            </svg>
+                            {{ 'navigation.admin.search'|trans|desc('Search') }}
+                        </a>
+                        <button class="btn btn-dark btn-block btn--udw-browse" data-starting-location-id="2">
+                            <svg class="ez-icon ez-icon-browse">
+                                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#browse"></use>
+                            </svg>
+                            Browse
```

Comment:
Translation?

Reply:
It's going to be fixed in #26, @mikadamczyk can leave this as-is since he only moved the code.